### PR TITLE
LPS-105689 View active changelist regardless of context in DXP

### PR DESCRIPTION
**Pull request by stsquared99**

### CODE REVIEW

File: modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/theme/contributor/ChangeTrackingIndicatorTemplateContextContributor.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+				themeDisplay.getCompanyId(), 0);
+
+		if (ctPreferences == null) {
+			return;
+		}
+
+		contextObjects.put(
+			"bodyCssClass",
+			GetterUtil.getString(contextObjects.get("bodyCssClass")) +
+				" has-change-tracking-indicator");
```
Comment (jbalsas) [inline modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/theme/contributor/ChangeTrackingIndicatorTemplateContextContributor.java]: DO we still need this?

Comment (stsquared99) [reply modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/theme/contributor/ChangeTrackingIndicatorTemplateContextContributor.java]: My bad, that should be removed

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/dynamic_include/change_tracking_indicator.jsp [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+		Liferay.detach('destroyPortlet', onDestroyPortlet);
+		Liferay.detach(
+			'<%= changeTrackingIndicatorDisplayContext.getEventName() %>',
+			onSelectChangeList
+		);
+	};
+
+	Liferay.on('destroyPortlet', onDestroyPortlet);
+
+	var onSelectChangeList = function() {
```
Comment (jbalsas) [inline modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/dynamic_include/change_tracking_indicator.jsp]: Could we move this to the React component? That way we don't need to add JS in JSPs which is untestable and it won't get transpiled.

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/content/Language.properties [hunk 3]
```diff
@@ -48,6 +49,7 @@ no-change-list-has-been-published-yet=No change list has been published yet.
 no-changes-were-found=No changes were found.
 no-collisions=No Collisions
 no-processes-were-found=No processes were found.
+no-publications-were-found=No publications were found.
```
Comment (jbalsas) [inline modules/apps/change-tracking/change-tracking-web/src/main/resources/content/Language.properties]: Don't we need to regen lang?

**Comment by stsquared99** [comment]
@jbalsas The current positioning is not going to work, I've included a screenshot below. That was why I initially had the dynamic include extension points in the layout_header and portlet_header. It was ugly code wise, but that seemed to be the only way to position it correctly relative to the header (which is not actually centered on the overall page) after talking to Patrick and JMak. Can you please advise? Thanks!
Screen Shot 2020-02-06 at 2 39 41 PM

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/ealonso/liferay-portal/pull/2766. [See changes here.](https://github.com/jbalsas/liferay-portal/compare/3215cbe0a531~3...3215cbe0a531) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**